testbot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint1
    pid: {p: 100.0, i: 10.0, d: 0.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint2
    pid: {p: 100.0, i: 10.0, d: 0.0}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint3
    pid: {p: 100.0, i: 10.0, d: 0.0}

  # Velocity Controllers ---------------------------------------
  wheel1_controller:
    type: effort_controllers/JointVelocityController
    joint: wheeljoint_1
    pid: {p: 1.0, i: 0.0, i_clamp: 0.0}

  wheel2_controller:
    type: effort_controllers/JointVelocityController
    joint: wheeljoint_2
    pid: {p: 1.0, i: 0.0, i_clamp: 0.0}

  wheel3_controller:
    type: effort_controllers/JointVelocityController
    joint: wheeljoint_3
    pid: {p: 1.0, i: 0.0, i_clamp: 0.0}

  wheel4_controller:
    type: effort_controllers/JointVelocityController
    joint: wheeljoint_4
    pid: {p: 1.0, i: 0.0, i_clamp: 0.0}

  wheel5_controller:
    type: effort_controllers/JointVelocityController
    joint: wheeljoint_5
    pid: {p: 1.0, i: 0.0, i_clamp: 0.0}

  wheel6_controller:
    type: effort_controllers/JointVelocityController
    joint: wheeljoint_6
    pid: {p: 1.0, i: 0.0, i_clamp: 0.0}

  wheel7_controller:
    type: effort_controllers/JointVelocityController
    joint: wheeljoint_7
    pid: {p: 1.0, i: 0.0, i_clamp: 0.0}

  wheel8_controller:
    type: effort_controllers/JointVelocityController
    joint: wheeljoint_8
    pid: {p: 1.0, i: 0.0, i_clamp: 0.0}
